### DISCUSSION TITLE
How do I change the CheckedListBox rectangle box color?

### DISCUSSION THREAD

**Discussion by jonny-xhl** [dotnet/winforms#4591] [Q&A] [answered]
When I set the ForeColor property, but the color of the rectangle box did not change.
![image](https://user-images.githubusercontent.com/29520329/108322923-00ddc600-7201-11eb-9a3f-ac1ab5ac5f49.png)

**Comment by RussKie** [comment, marked as answer]
You can't. `CheckedListBox` is essentially a `ListBox` control with `ListBox.DrawMode = OwnerDrawFixed` and custom drawn items. If you want custom checkboxes you would need to render those manually similarly to how it is done in `CheckedListBox`.

All rendering is done in `OnDrawItem`, including drawing checkboxes, background and text:
https://github.com/dotnet/winforms/blob/a6e811841ba54720f3237565d73a1a11535599b2/src/System.Windows.Forms/src/System/Windows/Forms/CheckedListBox.cs#L519

**Reply by jonny-xhl** [reply]
OK, I'll have a try.
